Андрианов Артем, 924

rampartrange opened the pull request.

rampartrange committed:
first commit

rampartrange committed:
changes

rampartrange committed:
changes

rampartrange committed:
redundunt fles removed

rampartrange committed:
redundunt fles removed

rampartrange committed:
modifications

rampartrange committed:
added Profile Editing

rampartrange committed:
added User Tests

rampartrange committed:
added Post feature and new tests

rampartrange committed:
changed user template and routes

WokinShopash commented on app.py:
```diff
@@ -0,0 +1,11 @@
+
+
+if __name__ == '__main__':
+    app.run()
```
Переноса строки не хватает

WokinShopash commented on app_test.py:
```diff
@@ -0,0 +1,48 @@
+        self.assertTrue(user.check_password('test'))
+        self.assertFalse(user.check_password('test1'))
+        self.assertEqual(user.bio, 'I am test')
+        self.assertNotEqual(user.bio, 'Python sucks')
```
А вот щас обидно было

WokinShopash commented on app_test.py:
```diff
@@ -0,0 +1,48 @@
+
+
+class TestUserModel(unittest.TestCase):
+
```
Пустая строка лишняя

WokinShopash commented on app_test.py:
```diff
@@ -0,0 +1,48 @@
+
+
+class TestPostModel(unittest.TestCase):
+
```
Пустая строка лишняя

WokinShopash commented on application/forms.py:
```diff
@@ -0,0 +1,32 @@
+class PostCreateForm(FlaskForm):
+    title = StringField('Title', validators=[DataRequired()])
+    text = TextAreaField('Text', validators=[Length(min=0, max=256)])
+    submit = SubmitField('Submit')
```
перенос строки

WokinShopash commented on config.py:
```diff
@@ -0,0 +1,31 @@
+    SQLALCHEMY_DATABASE_URI = environ.get('DATABASE_URL') or \
+        'sqlite:///' + path.join(basedir, 'app.db')
+    SQLALCHEMY_ECHO = False
+    SQLALCHEMY_TRACK_MODIFICATIONS = False
```
Нужен перенос строки

WokinShopash commented on application/forms.py:
```diff
@@ -0,0 +1,32 @@
+
+class ProfileEditForm(FlaskForm):
+    username = StringField('Username', validators=[DataRequired()])
+    bio = TextAreaField('Bio', validators=[Length(min=0, max=264)])
```
Почему 264?

WokinShopash reviewed (changes requested):
Удали из репозитория __pycache__

rampartrange committed:
pep8 rules and added consts

rampartrange committed:
Python doesn't suck

rampartrange committed:
Delete User storage.db

rampartrange committed:
deleted __pycache__

rampartrange committed:
pulled request
Merge branch 'dev3' of https://github.com/rampartrange/MIPT-Python into dev3
 Please enter a commit message to explain why this merge is necessary,

rampartrange committed:
Update README.md

WokinShopash reviewed (approved):
Зачтено.
Бонусы: 0.5 Web, 0.5 DB

rampartrange committed:
Merge branch 'master' into dev3
